core-api/src/main/java/com/absmartly/sdk/ContextEventHandler.java

```diff
@@ -8,4 +8,5 @@
 
 public interface ContextEventHandler {
 	CompletableFuture<Void> publish(@Nonnull final Context context, @Nonnull final PublishEvent event);
+	void flushCache();
```

Comment:
This seems a bit out of place. The "cache" should be an implementation detail, right?

Reply:
Changed